jvm/core/src/main/scala/org/scalatest/concurrent/Futures.scala

```diff
@@ -472,7 +472,10 @@ trait Futures extends PatienceConfiguration {
      * @throws TestFailedException if the future is cancelled, expires, or is still not ready after
      *     the specified timeout has been exceeded
      */
-    def futureValue(implicit config: PatienceConfig, pos: source.Position): T = {
+    // SKIP-DOTTY-START 
+    def futureValue()(implicit config: PatienceConfig, pos: source.Position): T = {
```

Comment:
Why are empty parens added here? This is a breaking change to existing source code.

Reply:
@bvenners Ah, sorry that was a mistake, the intention was to keep the Scala 2 version as it is, and add it only for Scala 3.

Reply:
Hmm, is it needed for Scala 3?

Reply:
@bvenners Yes, if I remember correctly, that was needed to get our JavaFuturesSpec to build and green, in the following commit: 

https://github.com/scalatest/scalatest/pull/1966/commits/e32243bf9dd4dcdfd59bae7a29ccac209358b30b

The JavaFuturesSpec refused to compile, if I remember correctly it was confused about the implicit parameters.

Having said so, it was 3.0.0-M3, we can try again 3.0.0-RC1 I suppose.